### ISSUE TITLE
How to remove all particles manually?

### ISSUE THREAD

**Issue by lewiszlw** [djeedai/bevy_hanabi#128]
Hi,

I want to manually destroy particles instead of by lifetime, how can I do?

**Comment by djeedai** [comment]
Hi @lewiszlw,

You can't. 🎆 Hanabi is GPU driven, and having to upload a kill switch for every single particle each frame would defeat this and destroy performance. What are you trying to achieve?

**Comment by lewiszlw** [comment]
Now I try to despawn ParticleEffect to achieve this, but there are some issue thrown by hanabi.

**Comment by djeedai** [comment]
The right way to do this is either to deactivate the spawner (if you plan to reuse it later) or despawn the effect completely.

What errors do you see? Which version are you on?

**Comment by lewiszlw** [comment]
I tried to deactivate the spawner, but the particles will still exists until lifetime ends.

bevy_hanabi version is 0.5.2. The error message below(despawn the effect)
```
thread 'Compute Task Pool (0)' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: Broken table invariant: buffer=0 row=1', /Users/lewis/.cargo/registry/src/github.com-1ecc6299db9ec823/bevy_hanabi-0.5.2/src/render/mod.rs:1375:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /Users/lewis/.cargo/registry/src/github.com-1ecc6299db9ec823/bevy_tasks-0.9.1/src/task_pool.rs:273:45
```

**Comment by djeedai** [comment]
Yes sorry you're hitting #123 which is fixed on `main` but I need to release a new version on `crates.io` with that fix. I'll see if I can do that this week.

**Comment by djeedai** [comment]
@lewiszlw I've released v0.5.3 with the fix for #123 can  you please try again that should fix that panic on despawning.

**Comment by lewiszlw** [comment]
Verified and no panic any more! Thanks!
